evm/gas-service.js

```diff
@@ -105,23 +110,36 @@ async function getGasUpdates(config, env, chain, destinationChains) {
                 return null;
             }
 
-            const {
+            let {
                 destination_native_token: {
+                    symbol: srcSymbol,
                     token_price: { usd: srcTokenPrice },
                     decimals: srcTokenDecimals,
                 },
+                ethereum_token: {
+                    token_price: { usd: ethPrice },
+                },
             } = sourceFeeData;
 
-            const {
+            let {
                 base_fee_usd: baseFeeUsd,
                 destination_express_fee: { total_usd: expressFeeUsd },
                 destination_native_token: {
+                    symbol: destSymbol,
                     token_price: { usd: destTokenPrice },
                     gas_price_in_units: { value: gasPriceInDestToken },
                 },
                 execute_gas_multiplier: executeGasMultiplier = 1.1,
             } = destinationFeeData;
 
+            if (srcSymbol === 'ETH' && srcTokenPrice === 100) {
```

Comment:
Sounds like axelarscan should fix this to return the correct hardcoded price

Reply:
I think that is to make our testnet fees cheaper for developers.

Reply:
what i meant was that `srcTokenPrice` should be set to the hardcoded `ethPrice` since we're subsidizing, to prevent this sort of workaround

Reply:
`source_token.token_price` for ETH shouldn't be the mainnet value since we're using a different testnet value. Can you ask axelarscan to change that perhaps?

Reply:
Yeah but it's `destination_native_token. token_price` in both cases, because we normalized pair and `source_token` is always a reference chain.